charts/civil-ccd/values.preview.template.yaml

```diff
@@ -1,5 +1,8 @@
 civil-service:
   java:
+    imagePullPolicy: Always
+    image: 'hmctspublic.azurecr.io/civil/service:pr-1831'
+    ingressHost: ${SERVICE_FQDN}
```

Comment:
This will need to be reverted prior to merging but I think that's known.

Marking the review as "Request changes" to prevent merging by mistake with this value not reset.